src/Account/Entities/History.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace UKFast\SDK\Account\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property int $id
+ * @property int $contact_id
```

Comment:
I believe this should be camelCase

```suggestion
 * @property int $contactId
```